infrastructure/global-resources/azuredeploy.json

```diff
@@ -199,7 +199,7 @@
         },
         {
             "type": "Microsoft.KeyVault/vaults/secrets",
-            "name": "[concat(parameters('key_vault_name'), '/dbConnectionString')]",
+            "name": "[concat(parameters('key_vault_name'), '/databaseConnectionString')]",
```

Comment:
Shall we make this caps too (e.g.) DATABASE_CONNECTION_STRING?

Reply:
I have tried to keep the same name as existing builds pipeline so there are no Failures for builds when connecting to Key Vault. If we change the name to capital, we will have to change it at every location in the build and release pipeline.